compiler/src/iree/compiler/Dialect/Flow/Transforms/test/fold_unit_dims.mlir

```diff
@@ -91,3 +91,19 @@ module @no_fold_public {
 //      CHECK:   util.func public @no_fold_global_unit_dims
 //      CHECK:     %[[LOAD:.+]] = util.global.load @[[GLOBAL]] : tensor<1x32x1x1x64xf32>
 //      CHECK:     %[[COLLAPSE:.+]] = tensor.collapse_shape %[[LOAD]]
+
+// -----
+
+module @fold_stream_parameter{
+  util.global private mutable @glob1
+    = #stream.parameter.named<"module"::"glob1"> : tensor<1x1x10xf32>
```

Comment:
style nit: prefer real words and the canonical formatting of IR - makes it easier for others to read/search, reformat, and move between files